### DISCUSSION TITLE
Make Phoenix open previous files / folders by default

### DISCUSSION THREAD

**Discussion by kowalsk** [phcode-dev/phoenix#692] [Ideas]
I love Phoenix but I'd like it to open previous files / folders the way I left them when I i.e. restart the computer. Currently I have to re-select previous folders / files and often I simply can't remember where I left off last time.

**Comment by abose** [comment]
@kowalsk Thanks for raising the issue.
Phoenix does open the last opened state where it is left off as brackets. But if multiple phoenix windows are open, then the rembere part wont work currently as it doesn't know what tab to save (This is bug and will be fixed).

If you do not have multiple Phoenix tabs open and is stiff facing the issue, then can you provide a step-by-step walkthrough of how to reproduce the issue for me to look into it.
